Sources/GRPCHTTP2Core/Client/Connection/ClientConnectionHandler.swift

```diff
@@ -264,6 +266,30 @@ final class ClientConnectionHandler: ChannelInboundHandler, ChannelOutboundHandl
   }
 }
 
+extension ClientConnectionHandler {
+  struct LoopBoundView: @unchecked Sendable {
+    private let handler: ClientConnectionHandler
+    private let context: ChannelHandlerContext
+
+    init(handler: ClientConnectionHandler, context: ChannelHandlerContext) {
+      self.handler = handler
+      self.context = context
+    }
+
+    func keepaliveTimerFired() {
+      self.handler.keepaliveTimerFired(context: self.context)
+    }
+
+    func keepaliveTimeoutExpired() {
+      self.handler.keepaliveTimeoutExpired(context: self.context)
+    }
+
+    func maxIdleTimerFired() {
+      self.handler.maxIdleTimerFired(context: self.context)
+    }
```

Comment:
Should we add an `assertInEventLoop()` call to these methods just to make sure we're not messing something up?